Backend/dev/migrations/dynamic-offer-driver-app/0343-rental-feat.sql

```diff
@@ -0,0 +1,91 @@
+ALTER TABLE atlas_driver_offer_bpp.ride ADD COLUMN odometer_start_reading double precision;
```

Comment:
Please check all these migrations, most the changes of these migrations are not there.